code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -61,6 +61,7 @@
 
 /obj/machinery/gibber/RefreshParts()
 	var/gib_time = 40
+	meat_produced = 0
```

Comment:
It's weird, But I wouldn't fix what already works. gib_time is a new variable, and eventually transfers to gibtime. I have no IDEA why.